aries-site/src/examples/components/datatable/DataTableVerticalListExample.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  DataTable,
+  Heading,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import ServerGroups from '../../../data/mockData/serverGroups.json';
+
+/* This function displays an arr's values in a vertical list within a table 
+   cell, hiding all values from view that exceed the given maximum list 
+   length until the "View More" button is clicked, at which point the 
+   button label changes to "View Less". */
+const ViewMore = (arr, max) => {
+  const [viewMore, setViewMore] = useState(false);
+  return (
+    <Box>
+      {viewMore
+        ? arr.map(item => <Text>{item}</Text>)
+        : arr.map(item => <Text>{item}</Text>).slice(0, max)}
+
+      {arr.length > max && (
+        <Button
+          label={viewMore ? 'View less' : `View more (${arr.length - max})`}
+          onClick={() => setViewMore(!viewMore)}
+        />
+      )}
+    </Box>
+  );
+};
+
+const data = ServerGroups.groups;
+const max = 3;
+const columns = [
+  {
+    property: 'name',
+    header: 'Name',
+    render: datum => <Text truncate>{datum.name}</Text>,
+  },
+  {
+    property: 'groupName',
+    header: 'Group Name',
+    render: datum => <Text truncate>{datum.groupName}</Text>,
+    sortable: true,
+  },
+  {
+    property: 'servers',
+    // header: 'Servers',
+    header: 'UNDER CONSTRUCTION',
```

Comment:
Should this be switched back to "Servers" now?

Reply:
Yeah I just switched it back, didn't realize I still had it commented out.